site/styles/_nav.scss

```diff
@@ -313,6 +313,7 @@
     font-size: 10px;
     border-color: transparent;
     background: rgba($color: $body-color, $alpha: 0.4);
+    color: #fff;
```

Comment:
Okay hardcoded?

Reply:
The search bar is always dark so i think it's fine.